.pylintrc

```diff
@@ -66,7 +66,7 @@ disable=raw-checker-failed,
         redefined-builtin,
         arguments-differ,
         dangerous-default-value,
-        logging-fstring-interpolation
+        import-outside-toplevel,
```

Comment:
Something unexpected happened here. You probably kept a line that was supposed to be removed, and vice versa. As a result, we are getting a lot of warnings during `make lint`: https://github.com/google/slo-generator/actions/runs/3204729299/jobs/5236369774. You probably just need to restore the original value.

Reply:
Yeah I need to keep the `logging-fstring-interpolation` you said before I have to remove it because it's not related to this PR.

Should I keep it now! 👀

Reply:
Yeah I think I didn't understood your first change.

I'll fix it